components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationConstants.java

```diff
@@ -122,8 +122,8 @@ private IdentityApplicationConstants() {
     public static final String IS_API_BASED_AUTHENTICATION_ENABLED_DISPLAY_NAME = "Is API Based Authentication Enabled";
     public static final String IS_ATTESTATION_ENABLED_PROPERTY_NAME = "IsAttestationEnabled";
     public static final String IS_ATTESTATION_ENABLED_DISPLAY_NAME = "Is Client Attestation Enabled";
-    public static final String IS_APPLICATION_ACCESS_ENABLED_PROPERTY_NAME = "applicationEnabled";
-    public static final String IS_APPLICATION_ACCESS_ENABLED_DISPLAY_NAME = "Application Enabled";
+    public static final String APPLICATION_ENABLED_PROPERTY_NAME = "applicationEnabled";
```

Comment:
```suggestion
    public static final String APPLICATION_ENABLED_PROPERTY_NAME = "isApplicationEnabled";
```
lets make this consistent with other properties we have ex: [IS_MANAGEMENT_APP_SP_PROPERTY_NAME](https://github.com/wso2/carbon-identity-framework/blob/master/components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationConstants.java#L102)

Reply:
This is based on the discussion internally we had. Pointed you to that.

Reply:
This change should not have an impact to the rest api property name.